src/main/java/HW_1/Task_1.java

```diff
@@ -0,0 +1,10 @@
+package HW_1;
+
+public class Task_1 {
+
```

Comment:
I can't compile and run this code
![image](https://github.com/AntBelyi/ALevel_automation/assets/66726924/799b0614-e941-4589-a435-3fef1b35f280)